k8s/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-kv-deployment
+  labels:
+    app: minio-kv
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: minio-kv
+  template:
+    metadata:
+      labels:
+        app: minio-kv
+    spec:
+      containers:
+      - name: minio-kv
```

Comment:
Need an env-var for the `port` too.

Reply:
What should the env var be called? I don't see it in the README, only those 2 already present - MINIO_ACCESS_KEY and MINIO_SECRET_KEY.